packages/react-static-tweets/src/twitter.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { createContext, ReactNode, useContext } from 'react'
```

Comment:
in the future, try to do a `git mv` instead of renaming in vscode so the git history stays intact (not a big deal, but a nice to have).

this is what i use for vscode renames https://marketplace.visualstudio.com/items?itemName=ambooth.git-rename

Reply:
Didn't know about `git mv` and I was annoyed about this too, awesome! Good to know

Reply:
I tried to fix it but it didn't work. Oh well :man_shrugging: